build.gradle

```diff
@@ -20,20 +21,36 @@ allprojects {
     }
 
     dependencies {
+        // Scala version for all scala dependencies
+        def scalaVersion = '2.12'
+
         // Use Scala 2.12 in our library project
         implementation 'org.scala-lang:scala-library:2.12.7'
 
         // Use Scalatest for testing our library
         testImplementation 'junit:junit:4.12'
-        testImplementation 'org.scalatest:scalatest_2.12:3.0.5'
+        testImplementation "org.scalatest:scalatest_$scalaVersion:3.0.5"
```

Comment:
Why dobule quotes?

Reply:
For `$scalaVersion` substitution to work